README.md

```diff
@@ -73,6 +73,54 @@ export default translate(['defaultNamespace', 'anotherNamespace'])(TranslatableV
 
 ```
 
+__getWrappedInstance()__: allows you to access to the component instance, wrapped into `translate()`. 
+Only available if you pass `{ withRef: true }` to the `translate()` options.
+
+```javascript
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+
+class TranslatableView extends Component {
+  
+  foo() {
+    // do something important
+  }
+  
+  render() {
+    const { t } = this.props;
+    
+    return (
+      <div>
+        <h1>{t('keyFromDefault')}</h1>
+      </div>
+    )
+  }
+}
+
+export default translate(['defaultNamespace', 'anotherNamespace'], { withRef: true })(TranslatableView);
+```
+
+```javascript
+import React, { Component } from 'react';
+import ./TranslatableView;
+
+class App extends Component {
+
+  handleClick() {
+    this.refs.translatedView.foo();
```

Comment:
@piotr-cz yes...agree...that readme was changed meanwhile and the new docu does not contain that sample anymore: https://react.i18next.com/components/translate-hoc.html